lua/plainline/core.lua

```diff
@@ -34,19 +34,25 @@ end
 
 -- Takes a table of functions (as produced by get_ptable), calls them and
 -- formats their results into a string, using the separator
```

Comment:
I think it would be good to update this comment to reflect the fact that `mkstatus` now takes the whole config as an argument, not just the separator.

Reply:
Done